stock_orderpoint_automatic_creation/models/product.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.model
+    def create(self, vals):
+        result = super(ProductProduct, self).create(vals)
+        orderpoint_obj = self.env['stock.warehouse.orderpoint']
+        wh_obj = self.env['stock.warehouse']
+        if result.type != 'service':
```

Comment:
It's weird to also create an orderpoint for `consumable` products. Should this not be:

```
if result.type == 'product':
```